css/templates/land-see/page.css

```diff
@@ -351,6 +350,119 @@ amp-img.cover img {
   z-index: 10;
 }
 
-.ampstart-input input[type="email"].user-invalid {
+.land-see-subscribe-form input[type="email"].user-invalid {
   border: 2px solid var(--color-form-invalid);
 }
+
+/* Accordion */
+
+.land-see-accordion section {
+  margin-bottom: 10px;
+  position: relative;
+  overflow: hidden;
+}
+
+.land-see-accordion section > h3 {
+  text-transform: uppercase;
+  font-weight: bold;
+  min-height: 68px;
+  line-height: 68px;
+  outline: none;
+}
+
+.land-see-accordion section > h3:after {
+  content: '';
+  position: absolute;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M6.967 13.932V0M0 6.966h13.932'/%3E%3C/svg%3E");
+  height: 14px;
+  width: 14px;
+  right: 30px;
+  top: 28px;
+  cursor: pointer;
+}
+
+.land-see-accordion section[expanded] > h3:after {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 14 14'%3E%3Cpath fill='none' stroke='%23021425' stroke-width='2' stroke-miterlimit='10' d='M0 6.966h13.932'/%3E%3C/svg%3E");
+}
+
+.land-see-accordion section:before {
+  content: '';
+  position: absolute;
+  height: 168px;
+  left:0;
+  right: 0;
+  background-color: var(--color-accordion-color-base);
+}
+
+.land-see .i-amphtml-accordion-header { /* reset defaults */
+  cursor: default;
+  background-color: inherit;
+  padding-right: 0;
+  border: none;
+}
+
+.land-see-accordion section.interiors:before {
+  opacity: var(--opacity-100);
+}
+
+.land-see-accordion section.fashion:before {
+  opacity: var(--opacity-77);
+}
+
+.land-see-accordion section.design:before {
+  opacity: var(--opacity-54);
+}
+
+.land-see-accordion section.arts-culture:before {
+  opacity: var(--opacity-30);
+}
+
+/* Post */
+
+.land-see-post-item {
+  max-width: 196px;
+  text-align: left;
+}
+
+.land-see .animate {
+  opacity: 0;
+  transform: translateY(-10px);
+}
+
+.land-see-post-item.wide {
```

Comment:
wide is not a good class name 

you can either have 

`land-see-post-item-wide` or use media query (may be mediaquery is the riht thing here)

Reply:
This isn't used until a future PR, but will revise CSS to reflect your comments.

Reply:
For added context on some of my CSS class naming... I got ahead of the review game before our back and forth on the last few PRs, and noticed the growing CSS file size for land-see as I added features. So I actually backtracked on some of the CSS namespacing in my local and fork branches, and tried to keep class names short and sweet for that reason. My goal was to make more room for CSS additions. We have about 15KB before we hit our 50K limit.